api/src/tests/test_context.rs

```diff
@@ -74,7 +77,7 @@ pub fn new_test_context(test_name: &'static str) -> TestContext {
             ApiConfig::default(),
         ),
         rng,
-        root_keys,
+        root_key,
```

Comment:
shall we rename this key from root_key to mint_key? since the only thing it can do is really just mint

Reply:
Sure